requirements.txt

```
@@ -11,3 +11,11 @@ jupyter~=1.0.0
 matplotlib~=3.6.2
 seaborn~=0.12.1
 tqdm~=4.64.1
+mlflow==2.1.1
+mlflow-skinny==2.1.1
+azureml-mlflow==1.48.0
+cloudpickle==2.2.0
+black==22.12.0
+onnxruntime~=1.13.1
+onnx~=1.13.0
+pillow~=9.4.0
```

Brakuje https://pypi.org/project/azureml-core/
```suggestion
pillow~=9.4.0
azureml-core~=1.49.0
```
```
---> 26 from azureml.core import Workspace
ModuleNotFoundError: No module named 'azureml.core'
```

---

Dodane